GenerateToolingFeed/V0Format/V0FormatCliEntry.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GenerateToolingFeed
+{
+    public class V0FormatCliEntry
+    {
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public string OS { get; set; }
+
+        public string Architecture { get; set; }
+
+        public string downloadLink { get; set; }
+
+        public string sha2 { get; set; }
+
+        public string size { get; set; }
+
+        public string @default { get; set; }
+    }
+}
```

Comment:
Should we really be duplicating this? Can we not at least inherit?

Reply:
Same note applies to other types in this directory.

Reply:
This was done on purpose to avoid unintentionally carrying feed changes between formats.

Reply:
But this isn't actually a different format. Hence my concern with allowing any potential for drift between these two.

Reply:
Have removed all duplicate code